core/libraries/form_sections/strategies/validation/EE_Equal_To_Validation_Strategy.strategy.php

```diff
@@ -0,0 +1,60 @@
+<?php
+use EventEspresso\core\domain\services\factories\EmailAddressFactory;
+use EventEspresso\core\domain\services\validation\email\EmailValidationException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+
+/**
+ * Class EE_Email_Validation_Strategy
+ *
+ * @package               Event Espresso
+ * @subpackage            core
+ * @author                Mike Nelson
+ * @since                 4.6
```

Comment:
still need to tweak this phpdoc header

the class name in the comment is wrong and you're still crediting Michael :)

Reply:
Done!